metadata.txt

```diff
@@ -26,6 +26,8 @@ status=final
 # Optional items:
 
 changelog=
+    Version 4.2.4:
+    - Fix bug Exception reading keywods #4632
     Version 4.2.4
```

Comment:
4.2.4 is now duplicated ...

Reply:
oh noooo....... forget to change the number.

Thanks @Gustry , will make another PR

Reply:
Going to release 4.2.5? So you need to update it in many places anyway